### PR TITLE
Add Fame Helsinki logo

### PR DESCRIPTION
**Pull request by rfay**
## The Issue

New sponsor: Fame Helsinki!

### CODE REVIEW

**Comment by rfay** [comment]
@bmartinez287 would. you mind helping me with this? There are two problems
1. Build failure that I can't understand. I seem to have one every time I touch anything.
2. I imagine you want to do something with darklogo and squarelogo

Build log is attached in case you can't see it

[ddev-com-front-end.8f161f00-625c-4865-8963-a851378e8566.log](https://github.com/ddev/ddev.com/files/14780298/ddev-com-front-end.8f161f00-625c-4865-8963-a851378e8566.log)

File: src/featured-sponsors.json [hunk 1]
```diff
@@ -60,6 +60,15 @@
     "url": "https://b13.com/",
     "github": "b13"
   },
+  {
```
[the remark this answers is not in the document]
Comment (bmartinez287) [reply src/featured-sponsors.json:63]: looks like the same but that worked on mine, that's odd.

[the remark this answers is not in the document]
Comment (rfay) [reply src/featured-sponsors.json:63]: Thanks so much. It was "squareLogo" (correct) vs "squarelogo" (incorrect). Very difficult to see difference between the capital L and lowercase l in that context.

**Comment by bmartinez287** [comment]
cool, so it was the duplicate key. good catch.

**Comment by rfay** [comment]
No, wasn't duplicate key, it was misspelling of "darkLogo" using lower-case "l"

Your fix double-pushed the stanza though.

In the future, feel free to push into the branch/PR on these.

**Comment by bmartinez287** [comment]
we are missing the dark and square logos on this one though.

**Pull request merged**